### DISCUSSION TITLE
GitHub SMS 2FA broken

### DISCUSSION THREAD

**Discussion by bmelloniCN** [community/community#64742] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Today I got the notification about 2FA being required in 45 days.  So I went to set it up.  I very much like to avoid installing apps on my phone so I chose the SMS 2FA option.  The GitHub SMS message to my US number keeps failing to be delivered.  

As a verification I both sent SMS to myself, and used several other SMS 2FA apps and they all worked fine.

To add insult to injury, even though ours is a *paid corporate account*... when I tried to contact support all I got is a worthless electronic agent that solves nothing and did not give me an option to contact a human support person.

Any suggestions on how to get support from a human?

**Comment by mdtanvirahamedshanto** [comment]
Responce #1

**Comment by anairamlc** [comment]
Here are some suggestions to get support from a human:

1. Visit the GitHub Support page: Go to the GitHub Support page and search for relevant topics or keywords related to your issue. There might be articles or guides that can help you troubleshoot the problem.

2. Contact GitHub Support via email: If you couldn't find a solution on the support page, you can email GitHub Support directly. Provide a detailed explanation of the issue you're facing and include any relevant screenshots or error messages. This will help them understand your problem better.

3. Reach out on social media: Try reaching out to GitHub's official social media accounts such as Twitter or LinkedIn. Sometimes, companies respond faster to public inquiries on these platforms.

**Comment by ghostinhershell** [comment]
Hi @bmelloniCN! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
